core/src/auth_service/db_handler.bal

```diff
@@ -0,0 +1,85 @@
+import ballerina/config as conf;
+import ballerina/mongodb;
+
+// Mongodb configurations.
+mongodb:ClientConfig mongoConfig = {
+    host: conf:getAsString("DB_HOST"),
+    username: conf:getAsString("DB_USER_NAME"),
+    password: conf:getAsString("DB_PASSWORD"),
+    options: {sslEnabled: false, serverSelectionTimeout: 500}
+};
+mongodb:Client mongoClient = check new (mongoConfig);
+mongodb:Database mongoDatabase = check mongoClient->getDatabase("EnvironmentPlatform");
+mongodb:Collection usersCollection = check mongoDatabase->getCollection("users");
+mongodb:Collection ministryCollection = check mongoDatabase->getCollection("ministries");
+mongodb:Collection adminCollection = check mongoDatabase->getCollection("admins");
```

Comment:
Look @yashodgayashan, @sanjulamadurapperuma duplication as mentioned in https://github.com/yashodgayashan/environment_platform/issues/34.

Reply:
Yes. we add it in a different file and import. I will do it in the next PR